miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -66,6 +66,19 @@ proc.authenticate_account_origin
     # => [storage_offset, storage_size]
 end
 
+#! Authenticates that the invocation of a kernel procedure originates from the account context.
```

Comment:
I think this line needs to be updated.

Also, I think this is a temporary solution, right? If so, I'd explain this in the comments.